src/github/repository.ts

```diff
@@ -31,14 +32,15 @@ export const createRepositoryWebhookHandler = async (context: WebhookContext, gi
 		});
 
 		await subscription.update({ totalNumberOfRepos: (subscription.totalNumberOfRepos || 0) + 1 });
+		await findOrStartSync(subscription, context.log, "partial");
```

Comment:
I think you might want to:
1. Passin `false` as the `isInitialSync` flag, coz that flag means is the WHOLE subscription is a new sync, and will take the date as the final backfillSinceDate to DB and show on UI label. But this is just one repo. So its date shouldn't take over.
2. Maybe also see what date shall we pass in? Take the existing backfillSinceDate in the subscription? I haven't think about it yet.

Reply:
ahh your right, i did this before that change lol - I will update. It will take backfillSinceDate so its consistent